Diffusion-macOS/ControlsView.swift

```diff
@@ -277,7 +279,33 @@ struct ControlsView: View {
                             }
                         }.foregroundColor(.secondary)
                     }
-                                        
+
+                    DisclosureGroup(isExpanded: $disclosedPreview) {
+                        CompactSlider(value: $generation.previews, in: 0...25, step: 1) {
```

Comment:
Good point, I can have it automatically decrement based on the threshold of one of the two.

Reply:
On second thought, this might need a different PR. I would have suggested this addition to the steps slider:
```swift
                        }.onChange(of: generation.steps) { newValue in
                            if generation.previews > newValue {
                                generation.previews = newValue
                            }
                        }
```

But the slider UI appears to be able to get out of sync with the binding somehow if it's changed from somewhere besides the drag gesture.
<img width="240" alt="image" src="https://github.com/huggingface/swift-coreml-diffusers/assets/1981179/f593e3e0-c09e-4416-b455-f4bae9d4eb56">